R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -308,6 +308,21 @@ numCyl <- summarize(groupBy(carsDF, carsDF$cyl), count = n(carsDF$cyl))
 head(numCyl)
 ```
 
+`groupBy` can be replaced with `cube` or `rollup` to compute subtotals across multiple dimensions.
```

Comment:
minor: I wouldn't say replace because they are not functionally the same?
how about `use cube or rollup to compute subtotals across multiple dimensions.`